src/Assets.php

```diff
@@ -128,6 +128,8 @@ public static function get_wc_block_data( $settings ) {
 				'reviewRatingsEnabled'        => wc_review_ratings_enabled(),
 				'productCount'                => array_sum( (array) $product_counts ),
 				'attributes'                  => array_values( wc_get_attribute_taxonomies() ),
+				'isShippingCalculatorEnabled' => 'yes' === get_option( 'woocommerce_enable_shipping_calc' ),
+				'isShippingCostHidden'        => 'yes' === get_option( 'woocommerce_shipping_cost_requires_address' ),
```

Comment:
Might be clearer to wrap this logic in a function – I'm not sure what the convention is for converting settings to booleans for front end. Not a blocker, just feels like boilerplate that we could standardise. e.g. get_boolean_option() => returns true/false.

Reply:
Let's keep an eye on this, but I'm wary of premature abstraction. I usually follow the [rule of three](https://andrewbrookins.com/technology/the-rule-of-three/) before doing abstractions like this.

Reply:
Another option (that'd be trivial to implement here) is using php's `filter_var`:

```php
filter_var( get_option( 'woocommerce_enable_shipping_calc' ), FILTER_VALIDATE_BOOLEAN )
```

And actually just found out from Mike (via slack) that there is a `wc_string_to_bool` function that could also be used (but `filter_var` might be preferable because it will account for other truthy responses).

Reply:
`wc_string_to_bool` looks like the right fit to me :)

Reply:
do you think it’s a blocker?

Reply:
No - `filter_var` is all good from my point of view. (FYI I've requested changes on recent review because of issue in front end and validation error.)